Include/README.rst

```diff
@@ -0,0 +1,63 @@
+The Python C API
+================
+
+The C API is divided into three sections:
+
+1. ``Include/``
+2. ``Include/cpython/``
+3. ``Include/internal/``
+
+
+Include: Limited API and Stable ABI
+===================================
+
+``Include/`` contains the public Limited API and the Stable ABI.  The
+Stable ABI is defined by :pep:`384`.
```

Comment:
Can you please summarize in 1-2 sentences what the stable ABI is, and how it relates to the limited API? For someone not already familiar with the distinction it may be a bit much to have to read that PEP.

Reply:
Maybe it is sufficient to add a few words about what an ABI is and what an API is? "Stable" and "Limited" are self-explanatory, right? :)

Reply:
I've updated this section with eb050465ed0a8061b00631025d25073bf15438c2. AFAICT, the Limited API is a part of the Stable ABI, and as such defined by PEP 384.

Reply:
I'm not sure how much detail we should put into this README file. Perhaps a longer and more descriptive version could be put into the Developer's Guide.

Reply:
PTAL. I've stolen some wording from PEP 652.